### DISCUSSION TITLE
Where is application error logs stored? 

### DISCUSSION THREAD

**Discussion by B-Esmaili** [vercel/next.js#11654] [Help]
Some of my website's users reported a bug telling me they keep getting  `"An unexpected error has occurred"` error.I couldn't find any way to get exception detail from their devices.The only thing come to mind is checking application error log file to get stack trace, but couldn't find `nextjs` error log file either.could you please tell me how can i debug my application?

**Comment by timneutkens** [comment]
Error logs are output to stderr. If you are hosting on zeit.co you can go to `<yourdomain>/_logs`
